src/main/java/com/google/sps/servlets/TasksServlet.java

```diff
@@ -79,36 +68,15 @@ public void doGet(
     assert googleCredential != null
         : "Null credentials (i.e. unauthenticated requests) should already be handled";
 
-    // Get tasks from Google Tasks
     TasksClient tasksClient = tasksClientFactory.getTasksClient(googleCredential);
     List<TaskList> allTaskLists = tasksClient.listTaskLists();
 
-    // Initialize Tasks Response
-    List<Task> tasks;
-
-    String taskLists = request.getParameter("taskLists");
-    if (taskLists == null) {
-      tasks = getAllTasksFromAllTaskLists(tasksClient);
-    } else {
-      List<String> selectedTaskListIds = Arrays.asList(taskLists.split(","));
-      tasks = getAllTasksFromSpecificTaskLists(tasksClient, selectedTaskListIds);
+    List<Task> allTasks = new ArrayList<>();
```

Comment:
Hmm - this is going to be troublesome for the Assign panel. 

Instead of this, could we accept a taskListId as a query parameter, and return the tasks associated with that taskListId?

Reply:
For more clarity, I need to know that the tasks belonged to the Assign task list (this method, while efficient, trims off that information)